### PR TITLE
Create meeting sequence Step 2

### PR DESCRIPTION
**Pull request by anna-giang**
Prototype for Step 2 of the Create Meeting sequence.
Users will use this page to select a meeting duration, and to choose whether they want to propose times manually or via Google Calendar.

### CODE REVIEW

File: prototype/src/main/webapp/create-meeting-step2.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  </head>
+  <body>
+    <div>
+      <h1>Find a Meeting Time</h1>
+      <h2>Step 2</h2>
+      <form>
+        <div>
+          <label for="duration-mins">Meeting Duration (minutes)</label>
+          <!-- maximum meeting length = 1440 mins (24 hours/all-day) -->
+          <input type="number" name="duration-mins" min="1" max="1440" />
```
Comment (vivianho18) [inline prototype/src/main/webapp/create-meeting-step2.html:17]: Could be better if the user could input meeting duration as number of hours and minutes - maybe an alpha feature

**Review by vivianho18 — COMMENTED** [review]
Looks good :)

**Review by theresenieva — COMMENTED** [review]
Yup, lgtm

**Review by ghost — APPROVED** [review]

File: prototype/src/main/webapp/create-meeting-step2.html (hunk 1 above)
Comment (anna-giang) [reply prototype/src/main/webapp/create-meeting-step2.html:17]: Yep, noted :)

**Pull request merged**